package.json

```diff
@@ -5,7 +5,7 @@
   "license": "BSD-3-Clause",
   "main": "lib/loop54-js-connector-client.js",
   "scripts": {
-    "clean": "rm -rf ./lib",
+    "clean": "rm -rf ./lib && mkdir lib",
```

Comment:
Once reason the TC job failed was that the `/lib` directory is deleted and the `node replaceVersion.js > lib/loop54-js-connector.js` part of the script can't write a file in a directory that doesn't exist.

`npm run bundle` should be fine. But the piping at the script level now means that running `npm run build-connector` individually would fail if your don't have the `/lib` directory on your disk.

How much of an issue do you think is this?

Reply:
I don't know, but can't it easily be fixed by adding the `mkdir lib` to `build-connector` instead? Seems strange to have the clean target re-create the output dir.